lib/tapioca/generators/dsl.rb

```diff
@@ -172,7 +172,8 @@ def constantize(constant_names)
         unless unprocessable_constants.empty?
           unprocessable_constants.each do |name, _|
             say("Error: Cannot find constant '#{name}'", :red)
-            remove(dsl_rbi_filename(name))
+            filename = dsl_rbi_filename(name)
+            remove_file(filename) if File.file?(filename)
```

Comment:
Added this extra check here so that we don't attempt to delete a file that doesn't exist and match the current test output.

Reply:
Should we use `File.exist?` instead? What's the difference between that and `file?`?